common/src/main/resources/locales/en-gb.yml

```diff
@@ -219,4 +219,6 @@ locales:
   transferclaim_command_description: 'Transfer ownership of the claim you''re standing in.'
   unclaimall_command_description: 'Delete all of your claims.'
   transferpet_command_description: 'Transfer ownership of a tamed animal.'
-  claimban_command_description: 'Ban a player from a claim.'
\ No newline at end of file
+  claimban_command_description: 'Ban a player from a claim.'
+  claim_private_enabled: 'Claim is set to private.'
+  claim_private_disabled: 'Claim is set to public.'
```

Comment:
Let's do two things to these two new locale keys (in every locale file):

- move them in the locale file to below the keys for claim bans
- add color formatting in the standard color (the bluey/green one I use) by wrapping the text with `[brackets](#color) `

The new command description locale key can stay put.

Reply:
Moved and added the green color :)